styles.scss

```diff
@@ -219,31 +193,279 @@ body {
+                left: 100%;
+                width: 100%;
+            }
+
+            &-title--bottom, &-title--right {
+                padding: 1rem;
+                letter-spacing: 0.1rem;
+                margin-top: 0.5rem;
+                font-weight: 500;
+                text-transform: uppercase;
+            } 
+    
+            &-title--bottom {
+                color: $item-grey;
+            }
+    
+            &-title--right {
+                color: $black-default;
+            }
+    
+            &-description {
+                color: $item-grey;
+                padding: 1rem;
+            }
+    
+            &-list {
+                margin: 0;
+                padding: 0;
+                width: 100%;
+                list-style: none;
+                @include flexbox(column, center, flex-start);
+    
+                &--docs-list {
+                    width: 100%;
+                    list-style: none;
+                    margin-top: 1.5rem;
+                    padding: 0;
+                }
+            }
+    
+            &-item {
+                position: relative;
+                color: $black-default;
+                transition: background-color 0.3s ease, color 0.3s ease;
+                padding: 1rem;
+                margin: 0;
+                width: 100%;
+                &:hover {
+                    color: $primary-color;
+                    background-color: $bg-hover;
+                }
+    
+                &--has-submenu:hover {
+                    .navbar__submenu--right {
+                        opacity: 1;
+                        visibility: visible;
+                    }
+                }
+    
+                &--has-submenu-lowest:hover {
+                    .navbar__submenu--right-lowest {
+                        opacity: 1;
+                        visibility: visible;
+                    }
+                }
+    
+                &--docs-item {
+                    padding: 1rem;
+                    margin: 0;
+                }
+    
+                &--docs-item:hover {
+                    background-color: $bg-hover;
+    
+                    .navbar__submenu-item-box-description {
+                        color: $black-default;
+                    }
+                }
+    
+                &-box {
+                    @include flexbox(row, flex-start, center);
+                    padding: 0.5rem;
+                    margin: 0.5rem;
+    
+                    &-content {
+                        margin-left: 1.5rem;
+                    }
+    
+                    &-title {
+                        font-weight: 600;
+                        padding: 0.2rem;
+                    }
+    
+                    &-description {
+                        color: $item-grey;
+                        padding: 0.2rem;
+                    }
+                }
+            }
+    
+            &-angle-right-icon {
+                position: absolute;
```

Comment:
I bet some of this styles could be a part of new block - `.icon`

Reply:
That's true, since `position` and `right` with the same values are already in `__icon` element. I'll move it